src/service/boards.js

```diff
@@ -0,0 +1,3 @@
+import rest from "./rest";
+
+export const boards = async () => rest.get('/board');
```

Comment:
As for me too much small files. I think the files auth.js (where is `register` function actually), and login.js might be merged. The file register.js might be deleted.